### ISSUE TITLE
Playlist-less mode

### ISSUE THREAD

**Issue by tuncer** [cmus/cmus#532]
Should there be a playlist-less mode or where, when configured, there is always just one playlist, and in playlist view it would automatically switch to the track list, removing the need to press tab first?

I tried to use the queue view for this, but when you run `cmus-remote -n` the playing track is removed from the queue although it's still there, because you can `cmus-remote -r`, and it will play what's not displayed anymore in the queue.

**Comment by mahkoh** [comment]
>removing the need to press tab first?

I'm not sure what you mean by this. You only have to press tab once after starting cmus. Unless you press tab again the cursor will stay in the track window.

**Comment by tuncer** [comment]
When I start cmus and switch to the queue view, even if there's a single playlist, I have to press tab before I am in the list of tracks to navigate.

**Comment by mahkoh** [comment]
But only once.

**Comment by tuncer** [comment]
Correct, and I'm saying if there's no other playlist, then it could automatically focus the tracks of that list. Make sense?

**Comment by mahkoh** [comment]
That could be done but it would having the cursor start sometimes here, sometimes there sounds unintuitive.

**Comment by tuncer** [comment]
If there's no other playlist, then the list of playlist serves no actionable purpose, so I would find it intuitive if it auto-focused the list.

Alternatively, would it make sense to remember which playlist was selected last, and focus that list upon re-entering playlist view?

**Comment by mahkoh** [comment]
Another issue is that the playlist cannot be selected if it's empty.

**Comment by tuncer** [comment]
Thinking about it, remembering the previously selected playlist (position in the list of playlists) makes sense regardless of auto-focus, don't you think?

**Comment by mahkoh** [comment]
There is a system for that but it's not currently implemented for playlists.

**Comment by vlada-dudr** [comment]
My issue with playlist selector is, that it takes lots of space, therefore I don't see all the information about tracks in playlists, which is important for me. Effectively split view renders playlists view unusable for me. I guess it would be enough to add "toggle playlist list" command/option.

**Comment by DamianPereira** [comment]
@mahkoh You also have to press tab every time you clear the playlist and add new songs. Try this:

- Broser: Add an album to playlist
- Playlist View: Press Tab to access songs, select song
- Delete all songs from playlist, either one by one or by clear command
- Switch to Browser, add another album
- Switch to playlist, you have to press tab again

This is specially cumbersome when you try to bind `clear -p` which does nothing unless the playlist is focused (with tab). If I try to clear the playlist while focusing Browser view, and add a few albums, when I switch to playlist view I realize it did nothing because playlist was not focused.

Something that would solve my issues:
- Switch focus to inside of a playlist when songs are added from Browser/Library.
- Make clear command work on the selected playlist, even if not inside.

Also, being able to disable multiple playlist completely would be nice, I have no use for it, and the panel uses lots of space.
